connectors/sources/mysql.py

```diff
@@ -106,11 +99,44 @@ async def close(self):
         await self.connection_pool.wait_closed()
         self.connection_pool = None
 
+    def _validate_connection_fields(self):
+        """Validates whether user input is empty or not for connection string
+
+        Raises:
+            Exception: Configured keys can't be empty
+        """
+        connection_fields = ["host", "port", "user", "password", "database"]
+        empty_connection_fields = []
+        for field in connection_fields:
+            if not self.configuration[field]:
```

Comment:
what is this test exactly? are you testing for `None`, for ``,  for `false` ?
always be explicit in this kind of test. Do not assume the boolean transformation will get what you want.

Reply:
Done